README.md

```diff
@@ -86,7 +86,7 @@ You can pass either the major version (eg 2.12) or the full version (eg 2.12.7):
     ./gradlew -PscalaVersion=2.12 test
     ./gradlew -PscalaVersion=2.12 releaseTarGz
 
-### Running a task with all scala versions ###
+### Running a task with all scala *default* versions (2.11.x and 2.12.x) ###
```

Comment:
Nit:

"Running a task with all the scala versions enabled by default (2.11.x and 2.12.x)"

Reply:
Done.